cmd/config-change-trigger/main.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/test-infra/prow/apis/prowjobs/v1"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/errorutil"
+	"k8s.io/test-infra/prow/flagutil"
+	prowgithub "k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pjutil"
+	pjdwapi "k8s.io/test-infra/prow/pod-utils/downwardapi"
+
+	"github.com/openshift/ci-tools/pkg/config"
+	"github.com/openshift/ci-tools/pkg/diffs"
+	"github.com/openshift/ci-tools/pkg/rehearse"
+)
+
+func loadClusterConfig() (*rest.Config, error) {
```

Comment:
we are using this also in [ci-operator](https://github.com/openshift/ci-tools/blob/master/cmd/ci-operator/main.go#L506-L522). We can share it instead of duplicating it.